examples/pytorch/graphsage/experimental/train_dist.py

```diff
@@ -51,8 +52,9 @@ def sample_blocks(self, seeds):
 
         input_nodes = blocks[0].srcdata[dgl.NID]
         seeds = blocks[-1].dstdata[dgl.NID]
-        batch_inputs, batch_labels = load_subtensor(self.g, seeds, input_nodes, "cpu")
-        blocks[0].srcdata['features'] = batch_inputs
+        batch_inputs, batch_labels = load_subtensor(self.g, seeds, input_nodes, "cpu", self.load_feat)
+        if self.load_feat:
+            blocks[0].srcdata['features'] = batch_inputs
```

Comment:
if we don't load features, what data are used in message passing in the input layer?

Reply:
If we don't load features, nodes will be transductive nodes.